src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/strategies/FizzBuzzOutputStrategyI18nProxy.java

```diff
@@ -0,0 +1,39 @@
+package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.strategies;
+
+import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.Constants;
+import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.strategies.FizzBuzzOutputStrategy;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * @author zwk
+ * @date 2021/12/15
+ **/
+public class FizzBuzzOutputStrategyI18nProxy implements FizzBuzzOutputStrategy {
+
+    private final FizzBuzzOutputStrategy _fizzBuzzOutputStrategy;
+
+    public FizzBuzzOutputStrategyI18nProxy(final FizzBuzzOutputStrategy _fizzBuzzOutputStrategy) {
+        this._fizzBuzzOutputStrategy = _fizzBuzzOutputStrategy;
+    }
+
+    /**
+     * @author zwk
+     * @date 17:53 2021/12/15
+     * @param output
+     * @return void
+     **/
+    @Override
+    public void output(String output) throws IOException {
+        Properties properties = PropertiesLoaderUtils.loadAllProperties(Constants.I18N_LIB_PROPERTIES_PATH);
```

Comment:
According to the [Spring documentation](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/core/io/support/PropertiesLoaderUtils.html#loadAllProperties-java.lang.String-), this method always uses ISO-8859-1, for compatibility with the 1990s. Did you verify that this code works properly on Microsoft Windows, printing the intended messages instead of [mojibake](https://en.wikipedia.org/wiki/Mojibake)?

Reply:
I think this is a good idea.  Many of our users are in 1995, thought the majority of our users are in 2007.